_posts/2024-11-25-New-ESMValCore-release.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: ESMValCore v2.11.1
+date: 2024-11-25
+---
+
+## ESMValCore v2.11.1 released!
+
+A new version of ESMValCore (v2.11.1) has been released:
+[https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.11.1](https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.11.1)
+
+### Highlights:
+
+This is a bugfix release which enables lazy computations in more preprocessors and allows installing the latests version of various dependencies, including Iris ([v3.11.0](https://github.com/SciTools/iris/releases/tag/v3.11.0)).
```

Comment:
"latests" without "s"

Reply:
Thanks @LisaBock for spotting this . Just fixed the typo!